### PR TITLE
83 project set up scripts arent os agnostic

### PR DESCRIPTION
**Pull request by ZacharyLeahan**
I added "npm run restore-test-data-win" as "npm run restore-test-data" did not run in Windows.
"npm run restore-test-data-win" is os agnostic as it relies on a js script. This is tested on my windows machine.

This pull request is for the js script, package.json reference, and readme update.

Evidence of this running locally:
image
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,19 +50,32 @@ sudo apt-get install -y mongodb-org
...
+sudo mongod --dbpath ~/data/db
 ```
 
-Run a Mongo instance: 
+##### For Windows Users
+Run a Mongo instance:
 
 ```
-sudo mongod --dbpath ~/data/db
+mongosh mongodb://localhost:27017/pa-wildflower-selector
```
Comment (boutell) [inline README.md:75]: This doesn't run an instance of mongodb, you already did that by setting it up to "install mongod as a service." This opens the mongo shell, which is nice but not required for anything we do below I think. You could just say "for Windows, this happened automatically when you installed mongod as a service."

File: scripts/restore-test-data.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+const https = require('https');
```
Comment (boutell) [inline scripts/restore-test-data.js:1]: This is impressive, but you're working way harder than you have to here by using promise chains (calling `then()` and `catch()` functions in chains etc). Using promise constructors and chains is not necessary anymore. You should switch this up to use `async/await` so your code can read as a simple linear series of steps. It's a lot easier to maintain.

There's a pretty good tutorial here:

https://javascript.info/async-await

**Review by boutell — CHANGES_REQUESTED** [review]
The request for changes is mostly about the minor confusion in the README, my comment about your promise chain based code is more about taking it in a direction you'll find a lot easier to maintain in the long run but it's up to you.

**Comment by ZacharyLeahan** [comment]
This isn't needed since git shell works on windows.

**Pull request closed without merging**